exercises/say/example.go

```diff
@@ -10,38 +10,51 @@ var tens = []string{"ones", "ten", "twenty", "thirty", "forty",
 var scale = []string{"thousand", "million", "billion",
 	"trillion", "quadrillion", "quintillion"}
 
-func Say(n uint64) string {
+func Say(n uint64) (string, error) {
+	if n >= 1000000000000 {
+		return "", &errorString{"Value out of range."}
```

Comment:
@MatForsberg I'm curious about why introduce a new error type instead of using `errors.New('value out of range')`? Your new type is not carrying extra data. Also I believe that the string should be in lower case entirely. You can get more information in here: 
https://github.com/golang/go/wiki/Errors

Reply:
Thank you. I have made the suggested changes. I did things that way because I am new to go and that is how i learned to do errors.

Reply:
Good eye @cored thanks!

Reply:
Thanks @cored :+1: